python/py-zmq/Portfile

```diff
@@ -60,8 +61,11 @@ if {${name} ne ${subport}} {
 
     } else {
 
-        depends_lib-append  port:py${python.version}-setuptools \
-                            port:py${python.version}-packaging
+        depends_lib-append  port:py${python.version}-setuptools_scm \
+                            port:py${python.version}-packaging \
+                            port:py${python.version}-wheel \
+                            port:py${python.version}-cffi \
+                            port:py${python.version}-cython
```

Comment:
not all of these dependencies should probably be `depends_lib`, are you sure that are all needed at runtime as well? Typically things like `setuptools_scm`, `packaging`, `wheel`, `cython` are only needed as build dependencies (but admittedly I haven't verified this).

Reply:
I don't recall changing other than the version of this port, but I'll move them to build dependencies.

Reply:
I don't know - if you didn't make changes in this area of the Portfile I wouldn't have had an opportunity to comment, so... ;) But regardless, did you verify that this is the correct change or you just did what I suggested without actually looking into it?

Reply:
I haven't got a clue about this package and I don't really have the time to go digging through it's source code. I've tried to update all the packages so that jupyter works and nothing breaks. Looking at zmq setup.py and pyproject.toml again, I assume I added these requirements because they're listed in pyproject.toml. I guess when I changed it back to use setuptools (something broke somewhere) I forgot to remove the extra dependencies.